core/templates/dev/head/pages/collection_editor/collection-editor.mainpage.html

```diff
@@ -0,0 +1,213 @@
+  <script src="/templates/dev/head/pages/exploration_editor/ExplorationInitStateNameService.js"></script>
+  <script src="/templates/dev/head/pages/exploration_editor/ExplorationLanguageCodeService.js"></script>
+  <script src="/templates/dev/head/pages/exploration_editor/ExplorationObjectiveService.js"></script>
+  <script src="/templates/dev/head/pages/exploration_editor/ExplorationParamChangesService.js"></script>
+  <script src="/templates/dev/head/pages/exploration_editor/ExplorationParamSpecsService.js"></script>
+  <script src="/templates/dev/head/pages/exploration_editor/ExplorationStatesService.js"></script>
+  <script src="/templates/dev/head/pages/exploration_editor/ExplorationTagsService.js"></script>
+  <script src="/templates/dev/head/pages/exploration_editor/ExplorationTitleService.js"></script>
+  <script src="/templates/dev/head/pages/exploration_editor/ChangeListService.js"></script>
+  <script src="/templates/dev/head/pages/exploration_editor/AutosaveInfoModalsService.js"></script>
+  <script src="/templates/dev/head/pages/exploration_editor/ExplorationDataService.js"></script>
+  <script src="/templates/dev/head/pages/exploration_editor/AngularNameService.js"></script>
+  <script src="/templates/dev/head/pages/exploration_editor/ExplorationEditor.js"></script>
+  <script src="/templates/dev/head/pages/exploration_editor/RouterService.js"></script>
+  <script src="/templates/dev/head/pages/exploration_editor/ChangesInHumanReadableFormService.js"></script>
+
+  <script src="/templates/dev/head/services/ExplorationFeaturesService.js"></script>
+  <script src="/templates/dev/head/services/ExplorationHtmlFormatterService.js"></script>
+  <script src="/templates/dev/head/services/ExtensionTagAssemblerService.js"></script>
+  <script src="/templates/dev/head/services/GenerateContentIdService.js"></script>
+  <script src="/templates/dev/head/services/HtmlEscaperService.js"></script>
+  <script src="/templates/dev/head/services/IdGenerationService.js"></script>
+  <script src="/templates/dev/head/services/LocalStorageService.js"></script>
+  <script src="/templates/dev/head/services/SiteAnalyticsService.js"></script>
+  <script src="/templates/dev/head/services/ValidatorsService.js"></script>
+
+  <script src="/templates/dev/head/pages/state_editor/state_properties/StateEditorService.js"></script>
+
+  <script src="/templates/dev/head/pages/exploration_editor/editor_tab/SolutionVerificationService.js"></script>
+  <script src="/templates/dev/head/pages/exploration_editor/editor_tab/SolutionValidityService.js"></script>
+  <script src="/templates/dev/head/domain/classifier/AnswerClassificationResultObjectFactory.js"></script>
+  <script src="/templates/dev/head/domain/classifier/ClassifierObjectFactory.js"></script>
+  <script src="/templates/dev/head/pages/exploration_player/PredictionAlgorithmRegistryService.js"></script>
+  <script src="/templates/dev/head/pages/exploration_player/StateClassifierMappingService.js"></script>
```

Comment:
Don't include the directive, module and controller scripts of collection editor here since they are already loaded in base.html and this file extends base.html so these aren't needed.

Reply:
Removed.